tests/realtikvtest/pessimistictest/pessimistic_test.go

```diff
@@ -539,7 +539,7 @@ func TestOptimisticConflicts(t *testing.T) {
 	tk.MustExec("begin pessimistic")
 	// This SQL use BatchGet and cache data in the txn snapshot.
 	// It can be changed to other SQLs that use BatchGet.
-	tk.MustExec("insert ignore into conflict values (1, 2)")
+	tk.MustExec("select * from conflict where id in (1, 2, 3)")
```

Comment:
I think this query can be removed.

Reply:
May I know why the query is unnecessary? IMO, it mean to test the [snapshot cache](https://github.com/tikv/client-go/blob/master/txnkv/txnsnapshot/snapshot.go#L132). We cache (1, 2) here and don't want to see the following select-for-update (L547) returns the cached row. Although, it won't read from the cache even when we forget to invalidate the cache because it will read from lock cache firstly now (also, we won't reuse the snapshot with for-update-ts now). I think it's ok to keep the query since the test point still makes sense.